src/main/java/io/stargate/sgv2/jsonapi/api/request/DataApiRequestInfo.java

```diff
@@ -19,6 +19,12 @@ public class DataApiRequestInfo {
   private final Optional<String> cassandraToken;
   private final Optional<String> embeddingApiKey;
 
+  public DataApiRequestInfo(Optional<String> tenantId) {
```

Comment:
Maybe add javadoc indicating why this constructor needed? (for offline use)

Reply:
Updated in this commit [3bf31d2](https://github.com/stargate/data-api/pull/905/commits/3bf31d24b56730eb75c36b05152633df26111ef9)